app/javascript/gobierto_data/webapp/components/commons/Info.vue

```diff
@@ -25,15 +25,42 @@
     </div>
     <div class="pure-u-1-2">
       <div
+        id="gobierto-data-summary-header"
         class="gobierto-data-summary-header-description"
-        v-html="descriptionDataset"
+        v-html="compiledHTMLMarkdown"
       />
+      <template v-if="checkStringLength">
+        <transition
+          name="fade"
+          mode="out-in"
+        >
+          <template v-if="truncateIsActive">
+            <span
+              class="gobierto-data-summary-header-description-link"
+              @click="truncateIsActive = !truncateIsActive"
+            >
+              {{ seeMore }}
+            </span>
+          </template>
+          <template v-else>
+            <span
+              class="gobierto-data-summary-header-description-link"
+              @click="scrollDetail"
+            >
+              {{ seeLess }}
+            </span>
+          </template>
```

Comment:
Since you conditionally render just an `span`, you may remove the `templates` and add the directives `v-if`/`v-else` in the spans directly

Reply:
Done!